### ISSUE TITLE
Equilibrators 

### ISSUE THREAD

**Issue by lkapelevich** [RalphAS/IterativeRefinement.jl#2]
We never take the absolute value of the matrix elements in https://github.com/RalphAS/IterativeRefinement.jl/blob/979a07364f8ba34766d4e15a16caa5acbbfa20c1/src/infra.jl#L82 Was that intended?

**Comment by RalphAS** [comment]
No, it was a careless oversight; thanks for pointing it out.  Should be fixed in master now.

**Comment by lkapelevich** [comment]
Thanks! I'm not sure if you meant to keep/use the `abs1`s at the top of the function?
